Library/Formula/kjell.rb

```diff
@@ -15,15 +17,14 @@ def install
   end
 
   test do
-    require 'open3'
-    Open3.popen3("#{bin}/kjell") do |stdin, stdout, _|
+    require "open3"
+    Open3.popen3("#{bin}/kjell") do |stdin, _, _|
```

Comment:
Can probably just use `IO.popen` if we're not doing anything with stdout/stderr?